src/core_ct/core.py

```diff
@@ -281,3 +281,63 @@ def chunk(self, x1=0, y1=0, z1=0, x2=None, y2=None, z2=None) -> Core:
 
         new_core = Core(self.pixel_array[x1:x2, y1:y2, z1:z2], self.pixel_dimensions)
         return new_core
+
+    def shape(self) -> tuple[int, int, int]:
+        """
+        Get the pixel dimensions of the core scan.
+
+        Arguments:
+        ---------
+            none
+
+        Returns:
+        -------
+            The pixel dimensions of the core scan.
+        """
+        return self.pixel_array.shape
+
+    def dimensions(self) -> tuple[float, float, float]:
+        """
+        Get the dimensions of the scan in mm.
+
+        Arguments:
+        ---------
+            none
+
+        Returns:
+        -------
+            The dimensions of the scan in mm.
+        """
+        return tuple(
+            size * dimension
+            for size, dimension in zip(self.pixel_array.shape, self.pixel_dimensions)
+        )
+
+    def volume(
+        self, min_density: float = 0, max_density: float = float("inf")
+    ) -> float:
+        """
+        Approximates the core volume in mm.
+
+        Arguments:
+        ---------
+            min_density: minimum density for a voxel to be counted towards the volume
```

Comment:
Can we specify these are floats in the docstring too?

Reply:
Removed these arguments as per the comment thread that I had with @TheSecondReal0 left on the `volume` method.

> Talked about this in person. We decided on removing the min_density and max_density arguments and making this function reliant on the filter method.